server/Cargo.toml

```diff
@@ -6,15 +6,15 @@ edition = "2018"
 
 [dependencies]
 base64 = "0.11.0"
-postgres-openssl = "0.3.0"
+env_logger = "0.7.1"
+hyper = "0.13.5"
+lockbook_core = { path = "../core" }
+log = "0.4.8"
 openssl = { version = "0.10", features = ["vendored"] }
-tokio-postgres = "0.5.4"
-rust-s3 = "0.18.6"
+postgres-openssl = "0.3.0"
+rsa = { version = "^0.2", features = ["serde1"] }
+rust-s3 = { version = "0.22.3", features = ["path-style"] }
```

Comment:
note: new rust-s3 version required for connecting to custom URLs for testing, version includes breaking changes such as making everything async